pkg/cmd/pr/shared/params.go

```diff
@@ -218,13 +219,34 @@ func SearchQueryBuild(options FilterOptions) string {
 			Label:     options.Labels,
 			Mentions:  options.Mention,
 			Milestone: options.Milestone,
+			Repo:      []string{options.Repo},
 			State:     options.State,
 			Type:      options.Entity,
 		},
 	}
 	return q.String()
 }
 
+func splitKeywords(keywords string) []string {
+	ks := []string{}
+	sb := &strings.Builder{}
+	quoted := false
+	for _, r := range keywords {
+		if r == '"' {
```

Comment:
[According to ES docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#_reserved_characters), special characters in queries such as `"` lose their special meaning if they're prefixed by a backslash. So if we're parsing queries like ES does, maybe we should align it more closely to their parser? Or we could avoid parsing ES altogether.